\doxysection{Référence de la classe pandas.\+errors.\+Chained\+Assignment\+Error}
\hypertarget{classpandas_1_1errors_1_1ChainedAssignmentError}{}\label{classpandas_1_1errors_1_1ChainedAssignmentError}\index{pandas.errors.ChainedAssignmentError@{pandas.errors.ChainedAssignmentError}}


Graphe d\textquotesingle{}héritage de pandas.\+errors.\+Chained\+Assignment\+Error\+:
% FIG 0


Graphe de collaboration de pandas.\+errors.\+Chained\+Assignment\+Error\+:
% FIG 1


\doxysubsection{Description détaillée}
\begin{DoxyVerb}Warning raised when trying to set using chained assignment.

When the ``mode.copy_on_write`` option is enabled, chained assignment can
never work. In such a situation, we are always setting into a temporary
object that is the result of an indexing operation (getitem), which under
Copy-on-Write always behaves as a copy. Thus, assigning through a chain
can never update the original Series or DataFrame.

For more information on view vs. copy,
see :ref:`the user guide<indexing.view_versus_copy>`.

Examples
--------
>>> pd.options.mode.copy_on_write = True
>>> df = pd.DataFrame({'A': [1, 1, 1, 2, 2]}, columns=['A'])
>>> df["A"][0:3] = 10 # doctest: +SKIP
... # ChainedAssignmentError: ...
>>> pd.options.mode.copy_on_write = False
\end{DoxyVerb}
 

La documentation de cette classe a été générée à partir du fichier suivant \+:\begin{DoxyCompactItemize}
\item 
env/lib/python3.\+12/site-\/packages/pandas/errors/\mbox{\hyperlink{env_2lib_2python3_812_2site-packages_2pandas_2errors_2____init_____8py}{\+\_\+\+\_\+init\+\_\+\+\_\+.\+py}}\end{DoxyCompactItemize}
